playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -27,9 +27,8 @@
         embedded_etcd: "{{ groups.oo_etcd_to_config | default([]) | length == 0 }}"
         debug_level: "{{ openshift_master_debug_level | default(openshift.common.debug_level | default(2)) }}"
 
-- name: Backup etcd
-  include: ./etcd/backup.yml
-  when: not openshift_upgrade_skip_etcd_backup | default(false) | bool
+- name: Upgrade and backup etcd
+  include: ./etcd/main.yml
```

Comment:
Actually, at first i tried something like this and I think I came to the conclusion you cannot conditionally include a playbook? Tasks of a play can conditionally include task files however.

(slightly simplified)

```
- name: Backup etcd
  include: ./etcd/backup.yml
  when: embedded_etcd

- name: Upgrade etcd
  include: ./etcd/main.yml
  when: not embedded_etcd
```

However, I think we should consider it to be a requirement to upgrade etcd.